docs/Deploy/aws_ec2.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: "Deploying to AWS EC2"
+sidebar_label: "AWS EC2"
+sidebar_position: 2
+---
+
+# Deploying to Amazon EC2
+
+Before you get started, Conduit should not be listening on any ports that are accessible to the public Internet. In this guide will we'll set up a Conduit instance that can be reached through the public Internet via SSH. We'll start at the top with a fresh AWS Account. In the end, the topology will look like this:
+
+![AWS EC2 Topology](/images/guides/aws-ec2-deployment/aws_ec2_conduit_setup.png)
+
+## Create a new VPC
+
+Start by logging into the AWS console, in your region of choice, and navigate to the VPC service. Click the `Create VPC` button at the top of the page.
+
+![Create VPC](/images/guides/aws-ec2-deployment/create-vpc.png)
+
+We'll select the `Create VPC and More` option within the page. This option will create all the default networking needed to create a VPC that can route traffic internally to it and reach out to the internet. But, the VPC will not be able to accept connections from outside the VPC to it. You'll need to add at least one `NAT Gateway` to the VPC to make that possible. If you don't, you will not be able to access the instance via SSH.
+
+![Select NAT Gateway](/images/guides/aws-ec2-deployment/select-nat-gateway.png)
```

Comment:
Those screens are really nice for readers to follow along with the setup on AWS -- great addition! :sparkles: